htdocs/core/class/html.formpropal.class.php

```diff
@@ -136,4 +136,62 @@ public function selectProposalStatus($selected = '', $short = 0, $excludedraft =
         }
         print '</select>';
     }
+
+	/**
+	 *    Return combo list of all commercial of proposals
+	 *
+	 *    @param	string	$selected   	Preselected value
+	 *    @param    string  $htmlname       Name of select field
+	 *    @param	int		$short			Use short labels
+	 *    @param	int		$excludedraft	0=All status, 1=Exclude draft status
+	 *    @param	int 	$showempty		1=Add empty line
+	 *    @param    string  $mode           'customer', 'supplier'
+	 *    @return	void
+	 */
+	public function selectProposalCommercial($selected = '', $htmlname = 'propal_commercial', $short = 0, $excludedraft = 0, $showempty = 1, $mode = 'customer')
+	{
+		global $langs;
+
+		$prefix = '';
+		$prefix = "PropalStatus";
+
+		$sql = "SELECT rowid, lastname, firstname FROM " . MAIN_DB_PREFIX . "user";
+		$sql .= " WHERE rowid IN";
+		$sql .= " (SELECT fk_user FROM ".MAIN_DB_PREFIX."societe_commerciaux) AND statut = 1 ORDER BY firstname ASC";
```

Comment:
Table societe_commerciaux is a table to store link between a third party and a user to save who is commercial of a third party. There is absolutely no relation between proposals. So the name and position of method is wrong here.
Also, there is already a function to get a combo list of users.